CMakeLists.txt

```diff
@@ -143,6 +143,12 @@ if(BUILD_TESTING)
 endif()
 list(REMOVE_DUPLICATES qt_required_components)
 
+if("${Qt${PythonQt_QT_VERSION}_VERSION}" VERSION_GREATER_EQUAL "6.10.0")
```

Comment:
I would put this just above `list(REMOVE_DUPLICATES qt_required_components)` line.